greedy/homework6.tex

```diff
@@ -0,0 +1,225 @@
+\documentclass[12pt]{article}
+
+\usepackage{tikz}
+\usetikzlibrary{trees}
+\usepackage[]{algorithm2e}
+\usepackage{amsmath}
+\newcommand{\BigO}[1]{\ensuremath{\operatorname{O}\bigl(#1\bigr)}}
+
+\begin{document}
+\title{Homework 5}
+\author{Robbie McKinstry, Jack McQuown, Cyrus Ramavarapu}
+\renewcommand{\today}{12 September 2016}
+\renewcommand{\baselinestretch}{1.5}
+
+\maketitle
+
+\section*{Greedy Problems}
+\subsection*{Problem 7:}
+A greedy algorithm that will minimize evictions is to 
+select the page that will be accessed furthest in the future
+from the current time.\\\\
+For example, assuming $k=4$ let the current state of the system be the
+following:
+\begin{center}
+    \begin{tabular}{c|c|c|c|c|c|c|c}
+    Time & 1 & 2 & 3 & 4 & 5 & 6 & 7 \\
+    Input & A & C & D & A & B & A & A \\   
+    \end{tabular}
+    Fast Memory ($k=4$) 
+    \begin{tabular}{c|c|c|c}
+    E & D & F & B 
+    \end{tabular}
+\end{center}
+In this situation, the algorithm will evict page B, since
+it is not used until time 5.  Any other choice will result
+in an earlier eviction. 
+\subsubsection*{Proof by Exchange:}
+Let $Alg$ be the process by which the above algorithm operates.
+Assume there exists an input $I$ such that $Alg(I)$ is 
+incorrect.  Let $Opt(I)$ be the optimal result for $I$
+that agrees with the greatest number of steps with $Alg(I)$.
+$Alg(I)$ and $Opt(I)$ must have a first point of disagreement.
+Label this time $t_i$.\\\\
```

Comment:
Should this be `$t_{I}$`? I haven't seen that syntax before and I can't see the rendered output.